.appveyor.yml

```diff
@@ -9,6 +9,9 @@ init:
 
 image: Visual Studio 2019
 environment:
+  global:
+    CCACHE_DIR: "%APPVEYOR_BUILD_FOLDER%\\.ccache"
+    CCACHE_MAXSIZE: 2G
```

Comment:
I do not think we have that much to play with - AFAICT we only have 1GByte according to: https://www.appveyor.com/docs/build-cache/#cache-size-beta - perhaps we want to tweak the `APPVEYOR_CACHE_ENTRY_ZIP_ARGS` from the default of fast but large archives `-tzip -mx=1` that is mentioned in the next section of that document?

Reply:
Will do